tests/test_shuffle.py

```diff
@@ -1,12 +1,31 @@
 import dask
+import dask.dataframe as dd
+import pytest
 
 
-def test_shuffle_simple(small_client):
-    df = dask.datasets.timeseries(
-        start="2000-01-01", end="2000-12-31", freq="1s", partition_freq="1D"
+@pytest.mark.stability
+def test_shuffle_simple(small_client, s3_url_factory, s3_storage_options):
```

Comment:
I would actually consider having both tests around is valuable. The original test doesn't involve IO, nor parquet.

How aobut

test_shuffle_simple
and
test_shuffle_parquet